src/test/java/org/jboss/forge/arquillian/util/DependencyUtilTest.java

```diff
@@ -57,29 +57,38 @@ public void should_get_last_non_snapshot_version() {
 
     @Test
     public void should_get_versions_only_supported_by_chameleon() throws Exception {
+        final Container container =
+            createContainer("container", Identifier.TOMCAT.getArtifactID(), Identifier.TOMCAT.getName());
+        final String groupId = container.getGroupId();
+        final String artifactId = container.getArtifactId();
+
+        // Assume following dependencies are getting after calling dependencyFacet.resolveAvailableVersions
         List<Coordinate> deps = new ArrayList<>();
-        deps.add(DependencyBuilder.create().setVersion("4.1").getCoordinate());
-        deps.add(DependencyBuilder.create().setVersion("2.1").getCoordinate());
-        deps.add(DependencyBuilder.create().setVersion("7.1").getCoordinate());
-        deps.add(DependencyBuilder.create().setVersion("1.0-SNAPSHOT").getCoordinate());
+        deps.add(DependencyBuilder.create().setGroupId(groupId).setArtifactId(artifactId).setVersion("4.1").getCoordinate());
+        deps.add(DependencyBuilder.create().setGroupId(groupId).setArtifactId(artifactId).setVersion("2.1").getCoordinate());
+        deps.add(DependencyBuilder.create().setGroupId(groupId).setArtifactId(artifactId).setVersion("7.1").getCoordinate());
+        deps.add(DependencyBuilder.create().setGroupId(groupId).setArtifactId(artifactId).setVersion("1.0-SNAPSHOT").getCoordinate());
 
-        List<String> dep = DependencyUtil.toVersionString(deps,
-            createContainer(Identifier.TOMCAT.getArtifactID(), Identifier.TOMCAT.getName()));
+        List<String> dependenciesToEndUser = DependencyUtil.toVersionString(deps, container);
 
-        assertThat(dep).doesNotContain("2.1", "4.1");
-        assertThat(dep).contains("7.1");
+        assertThat(dependenciesToEndUser).doesNotContain("2.1", "4.1");
+        assertThat(dependenciesToEndUser).contains("7.1");
```

Comment:
we don't define version in container files. we are matching version expression defined in chameleon for tomcat. It means we are supporting tomcat 7.1 . you can find it [here](https://github.com/arquillian/arquillian-container-chameleon/blob/master/arquillian-chameleon-container-model/src/main/resources/chameleon/default/containers.yaml#L453)